### ISSUE TITLE
M1 compatibility

### ISSUE THREAD

**Issue by borut-t** [auth0/JWTDecode.swift#137]
### Describe the problem you'd like to have solved
Compatibility for M1 macs.

### Describe the ideal solution
Be able to compile on M1 macs for simulator.

## Alternatives and current work-arounds
Run Xcode on Rosseta or run directly on the device.

### Additional context
`Could not find module 'JWTDecode' for target 'x86_64-apple-ios-simulator'; found: arm64, arm64-apple-ios-simulator, at: /Users/borutt/Library/Developer/Xcode/DerivedData/Proj-grnngzblfjxuwueukfcgqtmlzqbq/Build/Products/Dev_Debug-iphonesimulator/JWTDecode.swiftmodule`

Using SPM.

**Comment by borut-t** [comment]
> Hi @borut-t, thank you for raising this. Are you still experiencing this issue? If so:
> 
> * Which package manager are you using?
> * Are you using any dependency that is vendored as a binary?

Hey @Widcket, the issue is still there. I'm using SPM.

**Comment by Widcket** [comment]
@borut-t unfortunately I don't have an M1 mac around to test with. Does the issue persist if you use `ARCHS[sdk=iphonesimulator*] = $(NATIVE_ARCH_ACTUAL)` in the affected library/libraries?

**Comment by poovamraj** [comment]
Hey @borut-t, We tried to reproduce this issue on an M1 Mac and we were not able to reproduce it while making Xcode run in both Rosetta and normal mode. Can you provide us with more information on how to reproduce this issue?

**Comment by borut-t** [comment]
Hey @poovamraj. I have your dependency in my project and for some reason doesn't compile on the simulator with the following error: `Could not find module 'JWTDecode' for target 'x86_64-apple-ios-simulator';`.

I was able to compile it without a problem on a clean test project. So I need to dig deeper in order to understand why this is happing on my project but not on a test one.

I'll let you know when I found out.
